setup.py

```diff
@@ -16,7 +16,7 @@
     'boto3>=1.15.0,<2',
     'botocore>=1.18,<2',
     'cloudpickle>=2.1.0,<3.0',
-    'Faker>=10,<15',
+    'Faker>=17,<19',
```

Comment:
Is there any reason why we should cap to 19 instead of 20?

From the [release timeline](https://pypi.org/project/Faker/#history) I see that Faker released 19.0.0 in July of this year and is still on this major version (currently 19.11.0). If our integration works, I'd suggest making the requirement `<20`.

CC @amontanez24 I assume Faker is using some variant of semver such that all versions in `x.*.*` have backwards compatibility

Reply:
I agree with bounding it to v20. Faker v19 has been around for a few months (July 2023) so there is confidence in that version's stability.